editor/index.native.js

```diff
@@ -0,0 +1 @@
+export { default as PlainText } from './components/plain-text';
```

Comment:
@hypest is it enough to make it work on mobile?

Reply:
Thanks for asking @gziolo , yes, that's enough to make it work on mobile. I wonder though, wdyt about "native-ize" `./components/index.native.js` instead of `editor/index.native.js`? Is the `components` module too web-specific you think?

Reply:
The difference between `editor` and `components` is that `components` is meant to be generic UI components independent from WP backend or the editor while the `editor` is for components used to build blocks/editor. 

PlainText and RichText can be bound to the editor state, that's wy they live in editor.

Reply:
Thanks for the insight @youknowriad , that's helpful!

In this particular case, I'm actually referring to the editor/components mobule, not the top-level components. If I understand it correctly, PlainText and RichText live in the submodule in this PR, the index.js of which I'm suggesting to "native-ize". So, editor/index.native.js will still import * from './components' but editor/components/index.native.js will only export PlainText for now. Does that make sense?

Reply:
Oh, Understood and actually, I don't know. I don't really know how your react-native setup works, I thought the most important thing would be the root module (`editor/index`) but yeah if you need to keep the rest of the `editor/index` as is. I can update to move this to `editor/components/index.native.js`

Reply:
> I thought the most important thing would be the root module (editor/index)

Glad you mention that so we can expand a bit. I guess the main premise for now is to use the `index.js`s that serve as muster points as a good opportunity for specialization/nativization. Not necessarily the root module ones though. All `index.js`s can fall under that pattern. Hope this makes sense.

> I can update to move this to editor/components/index.native.js

Yes, let's go with that, thanks! 🙇